scripts/entrypoint.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+#
+# Copyright (C) 2021 Bosch.IO GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# License-Filename: LICENSE
+
+set -euo pipefail
+
+if [[ -f /root/.ssh/id_rsa ]]; then
+  eval $(ssh-agent -s)
+  ssh-add /root/.ssh/id_rsa
```

Comment:
I'm wondering why this should be necessary. `id_rsa` is one of the default keys OpenSSH tries, so if we *just* want to authenticate with *that* key, there should be no need to run an agent at all. Can you please elaborate?

Reply:
Actually, we would override that key - and therefore have to explicitly add it.
But it would probably be better to use a different directory and not overwrite the existing one, I agree.